lib/hexpm/security_vulnerability/updater.ex

```diff
@@ -0,0 +1,141 @@
+defmodule Hexpm.SecurityVulnerability.Updater do
+  @moduledoc false
+
+  use GenServer
+
+  require Logger
+
+  alias Hexpm.Repository.Packages
+  alias Hexpm.SecurityVulnerability.Disclosures
+
+  @disclosure_download_url "https://osv-vulnerabilities.storage.googleapis.com/Hex/all.zip"
```

Comment:
Where did this URL come from? I didnt see it documented on the osv.dev website.

Reply:
https://google.github.io/osv.dev/data/#data-dumps